content/en/docs/instrumentation/java/libraries.md

```diff
@@ -23,3 +24,169 @@ some instrumentation libraries for your dependencies standalone. To find out
 which standalone instrumentation libraries are available, take a look at
 [this list](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks).
 Follow the instructions of each instrumentation library to set them up.
+
+## Example app
+
+The following example instruments an HTTP client application using library
+instrumentation which calls an HTTP server.
+
+You can use the dice example app as HTTP server from
+[Getting Started](/docs/instrumentation/java/getting-started/) or you can create
+your own HTTP server.
+
+### Dependencies
+
+Set up an environment in a new directory named `java-simple-http-client`. Inside
+the directory, create a file named `build.gradle.kts` with the following
+content:
+
+{{% alert color="info" %}} Please note that this example is built with Gradle,
+you might need to amend directory structure and pox.xml to run using Maven
+{{% /alert %}}
+
+{{< tabpane text=true >}} {{% tab Gradle %}}
+
+```kotlin
+plugins {
+  id("java")
+  id("application")
+}
+
+application {
+  mainClass.set("otel.SampleHttpClient")
+}
+
+sourceSets {
+  main {
+    java.setSrcDirs(setOf("."))
+  }
+}
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+    implementation("io.opentelemetry:opentelemetry-api:{{% param vers.otel %}}");
+    implementation("io.opentelemetry:opentelemetry-sdk:{{% param vers.otel %}}");
+    implementation("io.opentelemetry:opentelemetry-exporter-logging:{{% param vers.otel %}}");
+    implementation("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:{{% param vers.otel %}}");
+    implementation("io.opentelemetry.instrumentation:opentelemetry-java-http-client:{{% param vers.instrumentation %}}-alpha");
+}
+```
+
+{{% /tab %}} {{% tab Maven %}}
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-java-http-client</artifactId>
+    <version>{{% param vers.instrumentation %}}-alpha</version>
+  </dependency>
+</dependencies>
+```
+
+{{< /tab >}} {{< /tabpane>}}
+
+### Setup
+
+The following example shows how you can instrument external API calls using
+[Java HTTP client library](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/java-http-client/library):
+
```

Comment:
Perhaps mentioned that with Spring Boot the SDK (OpenTelemetry object) can be initialized with the OpenTelemetry starter: https://opentelemetry.io/docs/languages/java/automatic/spring-boot

Reply:
Sure @jeanbisutti